java/runtime/src/main/java/org/ray/runtime/util/BinaryFileUtil.java

```diff
@@ -0,0 +1,49 @@
+package org.ray.runtime.util;
+
+import com.google.common.base.Preconditions;
+
+import java.io.File;
+import java.io.IOException;
+import java.io.InputStream;
+import java.nio.file.Files;
+import java.nio.file.Paths;
+import java.nio.file.StandardCopyOption;
+
+public class BinaryFileUtil {
+  public static final String REDIS_SERVER_BINARY_NAME = "redis-server";
+
+  public static final String GCS_SERVER_BINARY_NAME = "gcs_server";
+
+  public static final String PLASMA_STORE_SERVER_BINARY_NAME = "plasma_store_server";
+
+  public static final String RAYLET_BINARY_NAME = "raylet";
+
+  public static final String REDIS_MODULE_LIBRARY_NAME = "libray_redis_module.so";
+
+  public static final String CORE_WORKER_JAVA_LIBRARY = System.mapLibraryName("core_worker_library_java");
+
+  public static String getFilePath(String destDir, String fileName) {
```

Comment:
How about returning a `File` object? 
We can copy the file to session dir on demand in this function. Because workers and drivers require different files.